File: member-service/src/main/java/com/hedvig/memberservice/commands/UpdateWebOnBoardingInfoCommand.java
package com.hedvig.memberservice.commands;

import lombok.Value;
import org.axonframework.commandhandling.TargetAggregateIdentifier;

@Value
public class UpdateWebOnBoardingInfoCommand {

  @TargetAggregateIdentifier
  private Long memberId;
  private String SSN;
  private String email;
}

File: member-service/src/main/java/com/hedvig/memberservice/commands/UpdateWebOnBoardingInfoCommand.kt
package com.hedvig.memberservice.commands

import org.axonframework.commandhandling.TargetAggregateIdentifier

data class UpdateWebOnBoardingInfoCommand(
    @TargetAggregateIdentifier
    val memberId: Long,
    val SSN: String?,
    val email: String?
)
